src/utils/normalise.ts

```diff
@@ -0,0 +1,4 @@
+import uts46 from 'idna-uts46';
+
+export const normalise = (name: string): string =>
```

Comment:
Please use https://github.com/MyCryptoHQ/MyCrypto/blob/0e0b2610ff994fdac59e9885eec7189f8c3b02a3/src/services/EthService/utils/normalize.ts and perhaps move that out. We don't want the duplication.